metricbeat/module/kubernetes/node/node.go

```diff
@@ -82,5 +89,7 @@ func (m *MetricSet) Fetch() (common.MapStr, error) {
 		return nil, err
 	}
 
+	m.enricher.Enrich([]common.MapStr{event})
+
```

Comment:
With current abstractions metric sets have basically the same `Fetch` (and `New`) implementation for all resources, maybe we could have an only metricset for them, parametrized with the eventMapper and the enricher.

Reply:
Yep, I had the same feeling, I think it should be done in a follow up PR

Reply:
Perhaps time to also convert it to v2 API.